charts/invenio/templates/NOTES.txt

```diff
@@ -9,3 +9,20 @@ DEPRECATION WARNING:
     removed in a future release.
 
 {{- end }}
+
+{{- if .Values.invenio.sentry.existing_secret}}
```

Comment:
```suggestion
{{- if .Values.invenio.sentry.existing_secret }}
```

Reply:
Unfortunately not. I think the Kubernetes extension in VSCode has support for syntax highlighting, not sure about linting. There are some other extensions too, but I have not used any of them.

I have explored many Vim plugins for Helm, they are all terrible. 😕